openid-4-verifiable-presentations-1_0.md

```diff
@@ -460,6 +477,43 @@ To use `client_id_scheme` values `entity_id`, `did`, `verifier_attestation`, `x5
 
 Other specifications can define further values for the `client_id_scheme` parameter. It is RECOMMENDED to use collision-resistant names for such values.
 
```

Comment:
Since we are introducing a major change to the flow, would it make sense to mitigate phishing the way it is described here https://github.com/openid/OpenID4VP/issues/65 as well?